protos/sigstore_bundle.proto

```diff
@@ -59,7 +59,7 @@ message VerificationData {
 }
 
 message Bundle {
-        // MUST be application/vnd.dev.sigstore.bundle+json;version=0.1
+        // MUST be application/vnd.dev.sigstore.bundle+json;version=0.2
```

Comment:
CC @kommendorkapten what are the expectations around versioning here?

I feel like the plan is to make a bunch of breaking changes to the bundle in the very near future, is it worth even versioning it? Should we add a release process?

Reply:
i think no versioning at this point, this isn't being used anywhere. break away!

Reply:
Agreed, ignore versioning for now, we are still in the early days.

What I think might be the next major release is when Rekor SET v2 is out, then we should drop the raw Rekor respone, as it will be much easier/deterministic to verify the SET from a client perspective.

Reply:
I'm not sure when is the right time to add versioning, but what if we change this version specifier to contain the string `alpha` to emphasize that it's unstable? e.g.`version=0.1-alpha`.

Reply:
When we see the first real usage of the bundle (which I think is the npm use-case) in the wild, I think we can consider it released. I think it's too early to mess with this _now_. Ok, I know that's a bit underspecified but when we see a stable development for npm, which I hope is pretty soon, we can cut a release.